developer-docs-site/sidebars.js

```diff
@@ -359,6 +358,51 @@ const sidebars = {
         "tutorials/build-e2e-dapp/handle-tasks",
       ],
     },
+    {
+      type: "category",
+      label: "Learn about the Aptos Indexer",
```

Comment:
for now, I'd probably still put this under advanced guides or integrate with aptos

agreed it probably belongs under infrastructure and nodes becomes more general...

Reply:
Imo that's buried a little too deep. Anyway it doesn't really matter if we're gonna move it again in a few weeks.